ras_backstage/controllers/collection_exercise_controller.py

```diff
@@ -189,3 +189,27 @@ def update_collection_exercise_period(collection_exercise_id, period):
         raise ApiError(url, response.status_code)
 
     logger.debug('Successfully updated collection exercise period', collection_exercise_id=collection_exercise_id)
+
+
+def create_collection_exercise(created_collection_exercise_details):
+    logger.debug('Creating new collection exercise')
+    header = {'Content-Type': "application/json"}
+    url = f'{app.config["RM_COLLECTION_EXERCISE_SERVICE"]}collectionexercises'
+
+    payload = {
+        "surveyId": created_collection_exercise_details['survey_id'],
+        "name": created_collection_exercise_details['survey_name'],
+        "userDescription": created_collection_exercise_details['user_description'],
+        "exerciseRef": created_collection_exercise_details['period']
+    }
+
+    response = request_handler('POST', url, json=payload, headers=header, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code == 404:
```

Comment:
Doesn't look like 404 is returned on a post collection exercise request
" Returns 201 Created if the resource is created
Returns 400 Bad Request
Returns 409 Conflict "?